includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -314,7 +318,6 @@ static function() {
 		add_action( 'admin_notices', [ __CLASS__, 'print_admin_notice' ] );
 		add_action( 'admin_action_' . self::VALIDATE_ACTION, [ __CLASS__, 'handle_validate_request' ] );
 		add_action( 'post_action_' . self::UPDATE_POST_TERM_STATUS_ACTION, [ __CLASS__, 'handle_validation_error_status_update' ] );
-		add_action( 'admin_menu', [ __CLASS__, 'add_admin_menu_new_invalid_url_count' ] );
 		add_filter( 'post_row_actions', [ __CLASS__, 'filter_post_row_actions' ], PHP_INT_MAX, 2 );
 		add_filter( sprintf( 'views_edit-%s', self::POST_TYPE_SLUG ), [ __CLASS__, 'filter_table_views' ] );
 		add_filter( 'bulk_post_updated_messages', [ __CLASS__, 'filter_bulk_post_updated_messages' ], 10, 2 );
```

Comment:
Is there a reason not to bail right after the `register_post_type()` above if `$show_in_menu` is false? AFAIK the hooks below are irrelevant if dev tools is disabled for the user.

Reply:
Even if developer tools are turned off, the user can still access the validation screens. It's just that the UI is not shown unless the user specifically goes to them. This will enable us to be able to show a site health test warning with a link to the Validated URLs screen, even without dev tools enabled. You can confirm that by being on a validation screen and then in another tab, turn off developer tools. Then reload the previous tab.

Reply:
Oh right, yes that makes sense.